int8 left_friction
int8 right_friction
